site/src/helpers/util.tsx

```diff
@@ -127,6 +127,10 @@ export const sortTerms = (terms: string[]) =>
     return yearB.localeCompare(yearA) || quartersOrdered[qtrB].localeCompare(quartersOrdered[qtrA]);
   });
 
+export const isTermAfter = (earlierTerm: string, laterTerm: string): boolean => {
```

Comment:
earlier and later kinda implies that the order is already known? sorry i know naming this kind of thing sucks, but I think something like `term` and `otherTerm` is the most standard thing you can do.

That way, it should read "is [term I am comparing] after otherTerm", and by that, it's obvious that `true` means `term` is after `otherTerm`. Does that make sense?